subjects/devops/orchestrator/audit/README.md

```diff
@@ -57,13 +57,13 @@ NAME                                           STATUS   ROLES    AGE    VERSION
 $>
 ```
 
-###### Does the cluster created by a Vagrantfile?
+###### Was the cluster created by a Vagrantfile?
 
-###### Does the cluster contains 2 nodes (master and agent)?
+###### Is the cluster contains two nodes (master and agent)?
```

Comment:
Here `Is` is wrong, `Does` was the good verb to be used

Reply:
done